pkg/controller/plan/kubevirt.go

```diff
@@ -1202,7 +1220,9 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 	}
 
 	// qemu group
-	fsGroup := int64(107)
+	qemuUser := int64(qemuGroup)
```

Comment:
That did not came out the way I have imagined it. :) Could we have two constants `qemuUser` and `qemuGroup` to avoid this confusing assignments? That way it will be clear that those have same value on purpose (not by coincidence) and it's not a bug or a some hack. Here you would have `fsGroup := qemuGroup` then.

Reply:
unfortunately i must assign it. it's problematic to use const as pointer.
and yes, i will do 2 consts with 107 and just assign here..